docs/concepts/tokens/decentralized-exchange/automated-market-makers.md

```diff
@@ -29,6 +29,12 @@ An AMM holds two different assets: at most one of these can be XRP, and one or b
 
 When users want to trade in the decentralized exchange, their [Offers](offers.md) and [Cross-Currency Payments](../../payment-types/cross-currency-payments.md) can automatically use AMMs to complete the trade. A single transaction might execute by matching Offers, AMMs, or a mix of both, depending on what's cheaper.
 
+{% admonition type="info" name="Note" %}
+
+You can determine if a `Payment` interacted with an AMM by checking for a [`RippleState`](../../../references/protocol/ledger-data/ledger-entry-types/ripplestate.md) ledger entry in the transaction metadata. A flag value of `16777216` indicates AMM liquidity was consumed.
```

Comment:
Isn't it the same for the OfferCreate transaction as well as the Payment transaction?

Reply:
Good catch, adding it in.